Zend/zend_portability.h

```diff
@@ -248,6 +248,14 @@ char *alloca();
 # define ZEND_ATTRIBUTE_ALLOC_SIZE2(X,Y)
 #endif
 
+#if __STDC_VERSION__ >= 202311L
```

Comment:
just from a very quick glance, you might also need to enable it for C++ >= 17 (especially it s originally a C++ thing).

Reply:
Insight like this was the reason I requested your review. Do you have the necessary preprocessor condition?